spec/requests/api/v1/order_spec.rb

```diff
@@ -196,4 +211,75 @@
       end
     end
   end
+
+  describe 'POST /v1/order/:id/pay' do
+    let(:menu) { FactoryGirl.create(:menu) }
+    let(:menu_item1) { FactoryGirl.create(:menu_item, name: 'First Item', price: 60) }
+    let(:menu_item2) { FactoryGirl.create(:menu_item, name: 'Second Item', price: 50) }
+
+    let(:order_with_menu_items) do
+      order_with_mi = FactoryGirl.create(:order)
+      order_with_mi.order_items.create(menu_item: menu_item1, quantity: 2, menu: menu)
+      order_with_mi.order_items.create(menu_item: menu_item2, quantity: 2, menu: menu)
+      order_with_mi.save
+      order_with_mi
+    end
+
+    let(:customer) do
+      Stripe::Customer.create({
+        email: order_with_menu_items.user.email,
+        card: stripe_helper.generate_card_token
+      })
+    end
+
+    let(:stripe_helper) { StripeMock.create_test_helper }
+
+    before { StripeMock.start }
+    after { StripeMock.stop }
+
+    it "does a successfull payment process" do
+      json_data = { stripeToken: customer.card }
+      post "/v1/orders/#{order_with_menu_items.id}/pay", json_data
```

Comment:
Between  unit tests (`it`) you mean? Yes.

inside the `it` scope:
`let(order_with_menu_items)` will create an order the first time and it will return the same order after.